### ISSUE TITLE
File conflict between [openvino] and [crashpad]

### ISSUE THREAD

**Issue by Osyotr** [microsoft/vcpkg#32307]
Both libraries produce `util.lib` when building with `x64-windows-static`.
Related PR: https://github.com/microsoft/vcpkg/pull/31751

**Comment by Cheney-W** [comment]
This issue could be reproduced with the latest vcpkg 2023-06-22-f19f3d9939100085fe2f7e41502d60b08e2083b6.
```
error: The following files are already installed in F:/vcpkg/installed/x64-windows-static and are in conflict with openvino:x64-windows-static
Installed by crashpad:x64-windows-static
lib/util.lib
```

**Comment by Cheney-W** [comment]
@ilya-lavrenov 
About this issue, can we resolve it by modifying the name [here](https://github.com/openvinotoolkit/openvino/blob/18505b99b72dd6dbf8e4533ea998aa04c0932e1f/src/common/util/CMakeLists.txt#L5)? If so, do you have any suggestions for a new name? Thanks in advance!

**Comment by ilya-lavrenov** [comment]
Sure, I have create an upstream patch https://github.com/openvinotoolkit/openvino/pull/18801

**Comment by Osyotr** [comment]
@Cheney-W please note that crashpad should also be patched. Otherwise it shadows system library `libutil.so`.

**Comment by Cheney-W** [comment]
Stay active

**Comment by ilya-lavrenov** [comment]
Could you please remove OpenVINO from name? The issue with this library is fixed

**Comment by Osyotr** [comment]
Fixed in #32773.
